pkg/controller/operators/catalog/operator.go

```diff
@@ -1439,6 +1445,10 @@ func (o *Operator) unpackBundles(plan *v1alpha1.InstallPlan) (bool, *v1alpha1.In
 		// Ensure that bundle can be applied by the current version of OLM by converting to steps
 		steps, err := resolver.NewStepsFromBundle(res.Bundle(), out.GetNamespace(), res.Replaces, res.CatalogSourceRef.Name, res.CatalogSourceRef.Namespace)
 		if err != nil {
+			if fatal := olmerrors.IsFatal(err); !fatal {
```

Comment:
```suggestion
			if fatal := olmerrors.IsFatal(err); fatal {
```
Looking at this again, I think here we need to look for the fatal error here and return early as well